lib/validators/censored_words_validator.rb

```diff
@@ -12,4 +12,16 @@ def validate_each(record, attribute, value)
       )
     end
   end
+
+  private
+
+  def escape_censored_words
```

Comment:
Actually, I think you're correct and my specs aren't right. Let me see and get back to you. I have another PR open than enforces alpha numeric characters. Obviously one of them is going to have to be closed at some point. :)